runtime/runtime/src/global_contracts.rs

```diff
@@ -227,3 +227,55 @@ fn forward_distribution_next_shard(
     }
     Ok(())
 }
+
+pub(crate) trait AccountContractStoreExt {
```

Comment:
optional nit: I got a bit confused by `Store` in `AccountContractStoreExt` since it wasn't immediately obvious to me why TrieUpdate is a store, so may be better to have a slightly different name for a trait, `AccountContractRuntimeExt` or `AccountContractTrieUpdateExt`.